### DISCUSSION TITLE
A small question about facet crease stiffness

### DISCUSSION THREAD

**Discussion by KaiXiao55** [edemaine/fold#35] [Q&A]
Dear author, 

Sorry for bothering again, I am not sure to put this question in [https://github.com/amandaghassaei/OrigamiSimulator/issues](url) since it seems like that one does not have a discussion tab.

I found an interesting thing about facet crease stiffness in two different structures. And it looks like these two cases may lead to contradictive results. 

Specifically, I tested a closed-loop structure (4-bar linkage) and an open-loop structure. While increasing the stiffness of facet-crease (I guess it increases the rigidity of face?), the closed-loop structure's squared-shape face becomes non-flat whereas the open-loop structure's face becomes flat.

![a issue about rigid face (facet crease stiffness)](https://user-images.githubusercontent.com/75075763/108220360-9cbff100-7171-11eb-8ff9-40d3cef227f9.png)

[open_loop.txt](https://github.com/edemaine/fold/files/5996457/open_loop.txt)
[closed-loop.txt](https://github.com/edemaine/fold/files/5996456/closed-loop.txt)

Are both results making sense?  or my FOLD file has some problem?

I will very much appreciate it if you reply.

Kai Xiao.

**Comment by edemaine** [comment]
Please post to https://github.com/amandaghassaei/OrigamiSimulator/discussions which now exists.

**Reply by KaiXiao55** [reply]
Thanks for the guide. Now I have re-posted the question.
